tests/auto/api.json

```diff
@@ -2944,6 +3206,39 @@
             "r:o:currentAccessPoint": "$ref:WirelessAccessPoint",
             "r:state": "$ref:NetworkDeviceState"
         },
+        "ZWaveNetwork": {
+            "homeId": "Uint",
+            "isBridgeController": "Bool",
+            "isPrimaryController": "Bool",
+            "isStaticUpdateController": "Bool",
+            "isZWavePlus": "Bool",
+            "networkState": "$ref:ZWaveNetworkState",
+            "networkUuid": "Uuid",
+            "serialPort": "String",
+            "waitingForNodeAddition": "Bool",
+            "waitingForNodeRemoval": "Bool"
+        },
+        "ZWaveNode": {
+            "deviceType": "$ref:ZWaveDeviceType",
```

Comment:
Is the enum really required? Are you using it anywhere in the app or would a QString description do the job too?